src/main/java/org/terasology/launcher/config/ConfigValidator.java

```diff
@@ -49,6 +49,7 @@ Config validate(final Config config) {
                 .gameConfig(gameConfig.rebuilder()
                         .maxMemory(validateMaxMemory(gameConfig))
                         .initMemory(validateInitMemory(gameConfig))
+                        .javaParam(validateJavaParam(gameConfig))
```

Comment:
oh you implemented it in both classes! :joy:

Reply:
I started here, and then realized it's not used... thought I would not hurt to live it in... 🤷‍♂️